rescue_unique_constraint.gemspec

```diff
@@ -18,7 +18,7 @@ Gem::Specification.new do |spec|
   spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
   spec.require_paths = ["lib"]
 
-  spec.add_dependency "activerecord", ">= 3.2", "< 8"
+  spec.add_dependency "activerecord", ">= 3.2", "< 8.1"
```

Comment:
Think we can simply drop the upper limit? Seems like it does more harm than good unless we happen to know a future version will be incompatible.